cirq-core/cirq/experiments/single_qubit_readout_calibration_test.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, p0: float, p1: float, seed: 'cirq.RANDOM_STATE_OR_SEED_LIKE'
         self.p0 = p0
         self.p1 = p1
         self.prng = cirq.value.parse_random_state(seed)
-        self.simulator = cirq.Simulator(seed=self.prng)
+        self.simulator = cirq.Simulator(seed=self.prng, split_untangled_states=False)
```

Comment:
Why split=False?

Reply:
Seems to be specific seed that passes this even when split=False, so doesn't mean that split=True should also pass with same seed.